shared/actions/login/index.js

```diff
@@ -44,7 +44,7 @@ export function navBasedOnLoginState () :AsyncAction {
 
     // No status?
     if (!status || !Object.keys(status).length || !extendedConfig || !Object.keys(extendedConfig).length ||
-      !extendedConfig.device) { // Not provisioned?
+      !extendedConfig.defaultDeviceID) { // Not provisioned?
```

Comment:
if you use the 'falsey' nature of strings doing a ! is better. else you also have to check for it being null else you can crash on a null dereference
```!extendedConfig.defaultDeviceID || extendedConfig.defaultDeviceID.length === 0```